icons/v8/v8-plain.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+  <path fill="#4889f4" d="m3.766 21.836 10.95 19.166h.046l3.103 5.475h7.4l9.03 17.152a30.53 30.53 0 0 1 10.117-16.01c-3.124-4.578-4.578-10.095-4.045-15.621l-5.115-10.162H3.766zm88.982 0-5.19 9.098a23.61 23.61 0 0 1-3.978 16.68 30.694 30.694 0 0 1 10.111 16.105l9.051-17.242h7.393l.412-.684 2.695-4.791h.04l10.952-19.166H92.748zM70.326 62.834 63.932 75.02l-6.547-11.98c-2.28 1.847-3.825 4.661-3.836 8.077v.002c0 5.75 4.7 10.451 10.451 10.451a10.445 10.445 0 0 0 10.451-10.453c0-3.223-1.495-6.262-4.049-8.228-.025-.02-.05-.036-.076-.055zm-15.935 37.357L64 117.664l9.602-17.437a30.523 30.523 0 0 1-9.6 1.51 30.686 30.686 0 0 1-9.611-1.546z"/>
+  <path fill="#4889f4" d="M93.365 64.342a30.237 30.237 0 0 0-10.404-16.633 23.273 23.273 0 1 0-42.096-15.675 23.204 23.204 0 0 0 4.175 15.675 30.049 30.049 0 0 0-10.404 16.565 29.912 29.912 0 0 0-.753 6.845c0 12.732 7.94 23.683 19.166 28.064a30.186 30.186 0 0 0 10.952 2.053 30.049 30.049 0 0 0 29.365-36.894zM64 25.942a9.583 9.583 0 1 1 0 19.166 9.583 9.583 0 0 1 0-19.166zm0 56.128c-6.024 0-10.952-4.928-10.952-10.952a10.952 10.952 0 0 1 17.66-8.625 10.883 10.883 0 0 1 4.244 8.625A10.952 10.952 0 0 1 64 82.07Z"/>
```

Comment:
Nice!
Now we only need to merge the paths since it's a plain icon. According to our [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards):
> - The plain and line versions (with or without wordmark) need to stay as simple as possible. They must have only one color and the paths should be united.

Since both of the paths start with `M` I'm pretty sure we can just merge the `d` arguments like this:

```suggestion
  <path fill="#4889f4" d="m3.766 21.836 10.95 19.166h.046l3.103 5.475h7.4l9.03 17.152a30.53 30.53 0 0 1 10.117-16.01c-3.124-4.578-4.578-10.095-4.045-15.621l-5.115-10.162H3.766zm88.982 0-5.19 9.098a23.61 23.61 0 0 1-3.978 16.68 30.694 30.694 0 0 1 10.111 16.105l9.051-17.242h7.393l.412-.684 2.695-4.791h.04l10.952-19.166H92.748zM70.326 62.834 63.932 75.02l-6.547-11.98c-2.28 1.847-3.825 4.661-3.836 8.077v.002c0 5.75 4.7 10.451 10.451 10.451a10.445 10.445 0 0 0 10.451-10.453c0-3.223-1.495-6.262-4.049-8.228-.025-.02-.05-.036-.076-.055zm-15.935 37.357L64 117.664l9.602-17.437a30.523 30.523 0 0 1-9.6 1.51 30.686 30.686 0 0 1-9.611-1.546z M93.365 64.342a30.237 30.237 0 0 0-10.404-16.633 23.273 23.273 0 1 0-42.096-15.675 23.204 23.204 0 0 0 4.175 15.675 30.049 30.049 0 0 0-10.404 16.565 29.912 29.912 0 0 0-.753 6.845c0 12.732 7.94 23.683 19.166 28.064a30.186 30.186 0 0 0 10.952 2.053 30.049 30.049 0 0 0 29.365-36.894zM64 25.942a9.583 9.583 0 1 1 0 19.166 9.583 9.583 0 0 1 0-19.166zm0 56.128c-6.024 0-10.952-4.928-10.952-10.952a10.952 10.952 0 0 1 17.66-8.625 10.883 10.883 0 0 1 4.244 8.625A10.952 10.952 0 0 1 64 82.07Z"/>

```